desktop-src/FileIO/maximum-file-path-limitation.md

```diff
@@ -38,9 +38,9 @@ To enable the new long path behavior, both of the following conditions must be m
 * The registry key `Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem\LongPathsEnabled (Type: REG_DWORD)` must exist and be set to 1. The key's value will be cached by the system (per process) after the first call to an affected Win32 file or directory function (see below for the list of functions). The registry key will not be reloaded during the lifetime of the process. In order for all apps on the system to recognize the value of the key, a reboot might be required because some processes may have started before the key was set.
 
 You can also copy this code to a `.reg` file which can set this for you, or use the PowerShell command from a terminal window with elevated privileges:
-# [cmd](#tab/cmd)
+# [registry](#tab/registry)
```

Comment:
Thanks for your contribution. For this line, rather than using lowercase 'registry' in the tab, perhaps soemthing like this:
Registry (.reg) file